### DISCUSSION TITLE
is LCM Lora possible?

### DISCUSSION THREAD

**Discussion by habanerogossip** [PixArt-alpha/PixArt-alpha#26] [Ideas]
is Latent Consistency Model Lora possible for this? https://huggingface.co/blog/lcm_lora

**Comment by lawrence-cj** [comment]
Of course, It is. In my opinion, you may need just to train a specific LoRA(or other PEFT for Transformer) on Pixart-α.

**Reply by habanerogossip** [reply]
awesome. Does diffusers support lora training on pixart already or do you guys still need to implement it?

**Reply by lawrence-cj** [reply]
PixArt-LCM will be released soon. Stay tuned plz.

**Comment by yujincheng08** [comment]
PixArt-LCM released.
